packages/frontend/src/components/Home/LandingPanel/LandingPanel.styles.tsx

```diff
@@ -1,26 +1,6 @@
-import { Colors, H3 } from '@blueprintjs/core';
 import styled from 'styled-components';
 import LinkButton from '../../common/LinkButton';
 
-export const LandingHeaderWrapper = styled.div`
-    display: flex;
-    align-items: center;
-    padding-top: 60px;
-`;
-
-export const WelcomeText = styled.div`
-    flex: 1;
-`;
-
-export const Title = styled(H3)`
-    margin: 0 0 0.455em;
-    color: ${Colors.BLACK};
-`;
-
-export const Intro = styled.p`
-    color: ${Colors.GRAY1};
-`;
-
 export const StyledLinkButton = styled(LinkButton)`
     font-size: 14px !important;
 `;
```

Comment:
are we using this someplace else? if yes, can we get rid of it from this file and move where it's necessary?